astroid/brain/helpers.py

```diff
@@ -22,3 +22,112 @@ def transform(node: Module) -> None:
                     obj.parent = node
 
     manager.register_transform(Module, transform, lambda n: n.name == module_name)
+
+
+# pylint: disable-next=too-many-locals
+def register_all_brains(manager: AstroidManager) -> None:
+    from astroid.brain import (  # pylint: disable=import-outside-toplevel
```

Comment:
This seems like a good candidate for `from astroid.brain import *`, but I have a feeling you will prefer the explicitness.

Reply:
Yeah guessed correctly 😄 

Do you mind keeping the explicitness (for now)?

Reply:
Sure, that's fine. I'm still wondering if this should be in astroid.__init__ but I think there will have to be an import cycle to hide somewhere, and here is fine.

Reply:
Edited to say `astroid.brain.__init__`

Reply:
Oh, oops! Will fix tomorrow!

Reply:
This is going to need a refactor when we introduce conditional brain loading later (which is the point of the change, right ?). At this point we're either going to need to organize the brain under an enum of library (thinking ``{AstroidBrainLibrary.NUMPY: ["brain_numpy", "brain_numpy_core"]}``, and they're going to be explicitly sorted but we need to think about external brain registering if there is a pylint-numpy plugin with an external brain, or we'll filter them one way or another (thinking of something like "brain function contain the string 'numpy'" or "brain function match the regex *numpy*") and external brain will get auto-loaded. Probably better to argue about the final design than the tamporary one.

Reply:
Yeah, or we just move them into plugins? :smile: 

I don't know, removing this implicitness is an improvement either way.